src/test/integration/configuration-example.json

```diff
@@ -0,0 +1,23 @@
+{
+  "local_test": true,
+  "api": {
+    "local_base_url": "https://localhost:8080/api/v1",
+    "base_url": "https://api.oregonstate.edu/v1"
+  },
+  "auth": {
+    "basic_auth": {
+      "username": "username",
+      "password": "password"
+    },
+    "oauth2": {
+      "token_api_url": "https://api.oregonstate.edu/oauth2/token",
+      "client_id": "client_id",
+      "client_secret": "client_secret"
+    }
+  },
+  "test_cases": {
+    "valid_authority_onids": ["replace_with_valid_onid", "replace_with_valid_onid"],
+    "invalid_authority_onids": ["replace_with_no_authority_onid", "replace_with_no_authority_onid"],
+    "bad_request": [""]
```

Comment:
What should we put for `bad_request` field? If the only test case to generate a `400` is missing onid for `/spendingauthority` endpoint, I don't think it should be configurable, instead, you can hard code it in the testing section.